app/controllers/application_controller.rb

```diff
@@ -4,6 +4,8 @@ class ApplicationController < ActionController::API
   include ManageIQ::API::Common::ApplicationControllerMixins::RequestBodyValidation
   include ManageIQ::API::Common::ApplicationControllerMixins::RequestPath
 
+  BLACKLIST_PARAMS = [:tenant, :tenant_id]
```

Comment:
I don't think tenant_id needs to be here either because it isn't in the openapi doc so:
```
curl -X POST http://localhost:3000/api/v1.0/sources -d '{"source_type_id": "1", "name": "OCP", "tenant_id": "5678"}' 2>/dev/null | python -m json.tool
{
    "errors": [
        {
            "detail": "properties tenant_id are not defined in #/components/schemas/Source",
            "status": 400
        }
    ]
}
```